platform/masstransit.md

```diff
@@ -6,7 +6,7 @@ reviewed: 2025-01-23
 
 The Particular Service Platform provides error management capabilities for MassTransit endpoints on RabbitMQ and Azure Service Bus.
 
-It auto-detects and ingests messages from the [error](https://masstransit.io/documentation/concepts/exceptions#error-pipe) and [dead-letter](https://masstransit.io/documentation/concepts/exceptions#dead-letter-pipe) queues for all endpoints running in a MassTransit system.  
+It ingests messages from specified [error](https://masstransit.io/documentation/concepts/exceptions#error-pipe) and [dead-letter](https://masstransit.io/documentation/concepts/exceptions#dead-letter-pipe) queues for endpoints running in a MassTransit system.  
```

Comment:
I am not sure about pointing to https://masstransit.io/documentation/concepts/exceptions#dead-letter-pipe given that the content in that page talks about `_skipped` queues!